_posts/2022-06-08-packaging-functions-with-helm.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "Packaging OpenFaaS functions with Helm"
+description: "Learn how to package and deploy your OpenFaaS functions with Helm."
+date: 2022-06-08
+image: /images/2022-06-packaging-functions-with-helm/background.jpg
+categories:
+- helm
+- functions
+- kubernetes
+- enterprise
+author_staff_member: han
+---
+
+Learn how to package and deploy your OpenFaaS functions with Helm.
+
+## The OpenFaaS Operator
```

Comment:
I think we need to talk about "The OpenFaaS Operator", but it doesn't necessarily make sense as the opening heading.

Perhaps let's have a short introduction paragraph saying what they'll get out of reading the article, maybe why it's being written?

By short, I mean brief, short, not long. Easily digestible.